dts/st/f0/stm32f030c6tx-pinctrl.dtsi

```diff
@@ -304,54 +304,52 @@
 
 			usart1_rts_pa1: usart1_rts_pa1 {
 				pinmux = <STM32_PINMUX('A', 1, AF1)>;
-				bias-pull-up;
 				drive-open-drain;
 			};
 
 			usart1_rts_pa12: usart1_rts_pa12 {
 				pinmux = <STM32_PINMUX('A', 12, AF1)>;
-				bias-pull-up;
 				drive-open-drain;
 			};
 
 			/* UART_RX / USART_RX / LPUART_RX */
 
 			usart1_rx_pa3: usart1_rx_pa3 {
 				pinmux = <STM32_PINMUX('A', 3, AF1)>;
+				bias-pull-up;
```

Comment:
Pinmux is now configured straight after clock initialization, https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/serial/uart_stm32.c#L1318. I wonder what is the rationale behind this change, is it causing some sort of line toggling? Maybe driver should initialize pins even before clock init?

Reply:
When you are connected to external board, the line can be in undefined state when the external board is in reset state or during initialization. If there is no pull up, then some faulty characters can be inserted or overrun error may be occurred. Most hardware designs put buffers or external pull ups, but if it is not the case then the internal pull up should be set.

Reply:
If defaults do not work for a particular setup they can be overridden at board level, e.g.
```
&usart1_rx_pa3 {
    bias-pull-up;
};
```